src/main/java/frc/robot/lib/states/Arm.java

```diff
@@ -37,4 +74,62 @@ public void moveArmPosition(double firstStage, double secondStage, double wrist)
         RobotMap.armSecondStage.set(TalonFXControlMode.Position, secondStage);
         RobotMap.wrist.set(TalonFXControlMode.Position, wrist);
     }
+
+    /* getClosedLoopError(): Gets the closed-loop error. The units depend on which control mode is in use. 
+    If closed-loop is seeking a target sensor position, closed-loop error is the difference 
+    between target and current sensor value (in sensor units. 
+    Example 4096 units per rotation for CTRE Mag Encoder). */
+
+    public boolean getArrived(double allowance) {
+
+        // TODO: test and change error allowance
+        if (Math.abs(RobotMap.armFirstStage.getClosedLoopError()) <= Math.abs(allowance) && 
+            Math.abs(RobotMap.armSecondStage.getClosedLoopError()) <= Math.abs(allowance) &&
+            Math.abs(RobotMap.wrist.getClosedLoopError()) <= Math.abs(allowance)) { 
+            counter++;
+            if (counter > 15.0) {
```

Comment:
The pass threshold for the counter should also be configurable. It should also be time based, not a strict counter.